features/activity_runtime_api/publish.feature

```diff
@@ -333,7 +323,6 @@ Feature: External Activities can support a REST publishing api
       | launch_url      | http://activity.com/activity/1/sessions/ |
       | author_url      | http://activity.com/activity/1/edit |
       | print_url       | http://activity.com/activity/1/print_blank |
-      | description     | This activity does fun stuff. |
```

Comment:
I think it makes sense to leave the description in the example publishing data above. And then in the test you can verify that the description is ignored. This is because LARA will continue publishing the description for a while since we need to support the old portals.